bluemix-flame-detect/README.md

```diff
@@ -0,0 +1,35 @@
+# Bluemix flame detect Sample
+
+
+## Introduction
+This sample demonstrates using IBM® Bluemix™ to store information gathered be a sensor in a 
+clould service.
```

Comment:
typo 'clould'. Sentence is bit confusing and can be interpreted as if the sensor is in the cloud. Maybe rephrase it to end as 'gathered by a sensor.'